{
	"document": {
		"aggregate_severity": {
			"namespace": "https://www.suse.com/support/security/rating/",
			"text": "important"
		},
		"category": "csaf_security_advisory",
		"csaf_version": "2.0",
		"distribution": {
			"text": "Copyright 2024 SUSE LLC. All rights reserved.",
			"tlp": {
				"label": "WHITE",
				"url": "https://www.first.org/tlp/"
			}
		},
		"lang": "en",
		"notes": [
			{
				"category": "summary",
				"text": "Security update for python-Pillow",
				"title": "Title of the patch"
			},
			{
				"category": "description",
				"text": "This update for python-Pillow fixes the following issues:\n\n- CVE-2021-34552: Fixed a buffer overflow in Convert.c (bsc#1188574)\n",
				"title": "Description of the patch"
			},
			{
				"category": "details",
				"text": "SUSE-2021-2595,SUSE-OpenStack-Cloud-7-2021-2595",
				"title": "Patchnames"
			},
			{
				"category": "legal_disclaimer",
				"text": "CSAF 2.0 data is provided by SUSE under the Creative Commons License 4.0 with Attribution (CC-BY-4.0).",
				"title": "Terms of use"
			}
		],
		"publisher": {
			"category": "vendor",
			"contact_details": "https://www.suse.com/support/security/contact/",
			"name": "SUSE Product Security Team",
			"namespace": "https://www.suse.com/"
		},
		"references": [
			{
				"category": "external",
				"summary": "SUSE ratings",
				"url": "https://www.suse.com/support/security/rating/"
			},
			{
				"category": "self",
				"summary": "URL of this CSAF notice",
				"url": "https://ftp.suse.com/pub/projects/security/csaf/suse-su-2021_2595-1.json"
			},
			{
				"category": "self",
				"summary": "URL for SUSE-SU-2021:2595-1",
				"url": "https://www.suse.com/support/update/announcement/2021/suse-su-20212595-1/"
			},
			{
				"category": "self",
				"summary": "E-Mail link for SUSE-SU-2021:2595-1",
				"url": "https://lists.suse.com/pipermail/sle-security-updates/2021-August/009243.html"
			},
			{
				"category": "self",
				"summary": "SUSE Bug 1188574",
				"url": "https://bugzilla.suse.com/1188574"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2021-34552 page",
				"url": "https://www.suse.com/security/cve/CVE-2021-34552/"
			}
		],
		"title": "Security update for python-Pillow",
		"tracking": {
			"current_release_date": "2021-08-03T06:12:44Z",
			"generator": {
				"date": "2021-08-03T06:12:44Z",
				"engine": {
					"name": "cve-database.git:bin/generate-csaf.pl",
					"version": "1"
				}
			},
			"id": "SUSE-SU-2021:2595-1",
			"initial_release_date": "2021-08-03T06:12:44Z",
			"revision_history": [
				{
					"date": "2021-08-03T06:12:44Z",
					"number": "1",
					"summary": "Current version"
				}
			],
			"status": "final",
			"version": "1"
		}
	},
	"product_tree": {
		"branches": [
			{
				"branches": [
					{
						"branches": [
							{
								"category": "product_version",
								"name": "python-Pillow-2.8.1-4.25.1.aarch64",
								"product": {
									"name": "python-Pillow-2.8.1-4.25.1.aarch64",
									"product_id": "python-Pillow-2.8.1-4.25.1.aarch64"
								}
							},
							{
								"category": "product_version",
								"name": "python-Pillow-tk-2.8.1-4.25.1.aarch64",
								"product": {
									"name": "python-Pillow-tk-2.8.1-4.25.1.aarch64",
									"product_id": "python-Pillow-tk-2.8.1-4.25.1.aarch64"
								}
							}
						],
						"category": "architecture",
						"name": "aarch64"
					},
					{
						"branches": [
							{
								"category": "product_version",
								"name": "python-Pillow-2.8.1-4.25.1.ppc64le",
								"product": {
									"name": "python-Pillow-2.8.1-4.25.1.ppc64le",
									"product_id": "python-Pillow-2.8.1-4.25.1.ppc64le"
								}
							},
							{
								"category": "product_version",
								"name": "python-Pillow-tk-2.8.1-4.25.1.ppc64le",
								"product": {
									"name": "python-Pillow-tk-2.8.1-4.25.1.ppc64le",
									"product_id": "python-Pillow-tk-2.8.1-4.25.1.ppc64le"
								}
							}
						],
						"category": "architecture",
						"name": "ppc64le"
					},
					{
						"branches": [
							{
								"category": "product_version",
								"name": "python-Pillow-2.8.1-4.25.1.s390x",
								"product": {
									"name": "python-Pillow-2.8.1-4.25.1.s390x",
									"product_id": "python-Pillow-2.8.1-4.25.1.s390x"
								}
							},
							{
								"category": "product_version",
								"name": "python-Pillow-tk-2.8.1-4.25.1.s390x",
								"product": {
									"name": "python-Pillow-tk-2.8.1-4.25.1.s390x",
									"product_id": "python-Pillow-tk-2.8.1-4.25.1.s390x"
								}
							}
						],
						"category": "architecture",
						"name": "s390x"
					},
					{
						"branches": [
							{
								"category": "product_version",
								"name": "python-Pillow-2.8.1-4.25.1.x86_64",
								"product": {
									"name": "python-Pillow-2.8.1-4.25.1.x86_64",
									"product_id": "python-Pillow-2.8.1-4.25.1.x86_64"
								}
							},
							{
								"category": "product_version",
								"name": "python-Pillow-tk-2.8.1-4.25.1.x86_64",
								"product": {
									"name": "python-Pillow-tk-2.8.1-4.25.1.x86_64",
									"product_id": "python-Pillow-tk-2.8.1-4.25.1.x86_64"
								}
							}
						],
						"category": "architecture",
						"name": "x86_64"
					},
					{
						"branches": [
							{
								"category": "product_name",
								"name": "SUSE OpenStack Cloud 7",
								"product": {
									"name": "SUSE OpenStack Cloud 7",
									"product_id": "SUSE OpenStack Cloud 7",
									"product_identification_helper": {
										"cpe": "cpe:/o:suse:suse-openstack-cloud:7"
									}
								}
							}
						],
						"category": "product_family",
						"name": "SUSE Linux Enterprise"
					}
				],
				"category": "vendor",
				"name": "SUSE"
			}
		],
		"relationships": [
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python-Pillow-2.8.1-4.25.1.aarch64 as component of SUSE OpenStack Cloud 7",
					"product_id": "SUSE OpenStack Cloud 7:python-Pillow-2.8.1-4.25.1.aarch64"
				},
				"product_reference": "python-Pillow-2.8.1-4.25.1.aarch64",
				"relates_to_product_reference": "SUSE OpenStack Cloud 7"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python-Pillow-2.8.1-4.25.1.s390x as component of SUSE OpenStack Cloud 7",
					"product_id": "SUSE OpenStack Cloud 7:python-Pillow-2.8.1-4.25.1.s390x"
				},
				"product_reference": "python-Pillow-2.8.1-4.25.1.s390x",
				"relates_to_product_reference": "SUSE OpenStack Cloud 7"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python-Pillow-2.8.1-4.25.1.x86_64 as component of SUSE OpenStack Cloud 7",
					"product_id": "SUSE OpenStack Cloud 7:python-Pillow-2.8.1-4.25.1.x86_64"
				},
				"product_reference": "python-Pillow-2.8.1-4.25.1.x86_64",
				"relates_to_product_reference": "SUSE OpenStack Cloud 7"
			}
		]
	},
	"vulnerabilities": [
		{
			"cve": "CVE-2021-34552",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2021-34552"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "Pillow through 8.2.0 and PIL (aka Python Imaging Library) through 1.1.7 allow an attacker to pass controlled parameters directly into a convert function to trigger a buffer overflow in Convert.c.",
					"title": "Vulnerability description"
				}
			],
			"product_status": {
				"fixed": [
					"SUSE OpenStack Cloud 7:python-Pillow-2.8.1-4.25.1.aarch64",
					"SUSE OpenStack Cloud 7:python-Pillow-2.8.1-4.25.1.s390x",
					"SUSE OpenStack Cloud 7:python-Pillow-2.8.1-4.25.1.x86_64"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2021-34552",
					"url": "https://www.suse.com/security/cve/CVE-2021-34552"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1188574 for CVE-2021-34552",
					"url": "https://bugzilla.suse.com/1188574"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"SUSE OpenStack Cloud 7:python-Pillow-2.8.1-4.25.1.aarch64",
						"SUSE OpenStack Cloud 7:python-Pillow-2.8.1-4.25.1.s390x",
						"SUSE OpenStack Cloud 7:python-Pillow-2.8.1-4.25.1.x86_64"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 7.5,
						"baseSeverity": "HIGH",
						"vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
						"version": "3.1"
					},
					"products": [
						"SUSE OpenStack Cloud 7:python-Pillow-2.8.1-4.25.1.aarch64",
						"SUSE OpenStack Cloud 7:python-Pillow-2.8.1-4.25.1.s390x",
						"SUSE OpenStack Cloud 7:python-Pillow-2.8.1-4.25.1.x86_64"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2021-08-03T06:12:44Z",
					"details": "important"
				}
			],
			"title": "CVE-2021-34552"
		}
	]
}
